unbxdAutosuggest.js

```diff
@@ -163,6 +163,10 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 		}
 	};
 
+	var isDesktop = function() {
+		return !((this.options.isMobile && this.options.isMobile()) || isMobile.any());
```

Comment:
check if its a function:
```
if(typeof(this.options.isMobile) === "function")
```

Reply:
Instead it should be this?
`		return !(typeof(this.options.isMobile) === "function" ? this.options.isMobile() : isMobile.any());
`